CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.0.2]
```

Comment:
I released a patch because 10.0.0 was already marked as a breaking change for EIP-1993 compatibility.

Reply:
Makes sense to put this under a patch, since the changes shouldn't have any functional differences.